### DISCUSSION TITLE
Can I use the VR crosshair in non VR 3rd person mode to simulate bullets coming from barrel?

### DISCUSSION THREAD

**Discussion by eheinzg** [praydog/REFramework#869] [General Questions] [answered]
I was wondering if the VR crosshair can be used in regular third person for RE2/3 and 4. I tried removing the lines containing:

"if not vrmod:is_hmd_active() then return true end"

but it didn't work.  It would be cool to do use the VR crosshair since it realistically simulates bullets coming from the barrel of the gun. This could also be used to create an OG RE4 laser system. 

Is this possible? Thanks

**Comment by praydog** [comment, marked as answer]
It is unfortunately not that simple. The VR code in C++ is the one that makes the bullet come out of the muzzle instead of the center of the screen. In RE2/3 that is as simple as modifying a variable in the weapon configuration. In RE4 and the other games this is done by hooking the weapon firing function. You can make the 3D cross hair work outside of VR but will require some more in depth changes.

 I'm not currently in a position at the moment to work on anything due to some unfortunate circumstances.

**Reply by ErwinGunSmtih** [reply]
Hi @praydog, just following up from my other account. I found a parameter in the weapon catalog for RE4 that makes the bullets come from the muzzle. The parameter is called _GenerateFollowTarget and must be set to "Generator." Not sure if you already found this by now but it will be helpful for first person VR.
![image](https://github.com/praydog/REFramework/assets/160447491/830d1aff-b8fb-4f67-afd7-0308704ddc81)
